src/java/org/apache/cassandra/cql3/statements/schema/CopyTableStatement.java

```diff
@@ -248,7 +278,24 @@ public CQLStatement prepare(ClientState state)
         {
             String oldKeyspace = oldName.hasKeyspace() ? oldName.getKeyspace() : state.getKeyspace();
             String newKeyspace = newName.hasKeyspace() ? newName.getKeyspace() : state.getKeyspace();
-            return new CopyTableStatement(oldKeyspace, newKeyspace, oldName.getName(), newName.getName(), ifNotExists, attrs);
+            return new CopyTableStatement(oldKeyspace, newKeyspace, oldName.getName(), newName.getName(), ifNotExists, createLikeOption, attrs);
+        }
+
+        public void setLikeOptions(CreateLikeOption likeOption)
+        {
+            this.createLikeOption = likeOption;
+        }
+    }
+
+    public enum CreateLikeOption
+    {
+        EMPTY("EMPTY"),
```

Comment:
This is not used anywhere except in 

`private CreateLikeOption createLikeOption = CreateLikeOption.EMPTY;`

You can just set `createLikeOption` to `null`, you do not need `EMPTY`. You can remove `EMPTY`.